charts/sptribs-case-api/values.preview.template.yaml

```diff
@@ -92,7 +92,7 @@ ccd:
     java:
       disableKeyVaults: true
       environment:
-        CCD_DATA_STORE_URL: http://${SERVICE_NAME}-ccd-data-store-api
+        CCD_DATA_STORE_URL: http://sptribs-case-api-pr-${CHANGE_ID}-ccd-data-store-api
```

Comment:
NB: the lines above appear wrong to me: please re-review

```
  am-role-assignment-service:
      ingressHost: ccd-data-store-api-${SERVICE_FQDN}
  am-role-assignment-service:
    java:
      disableKeyVaults: true
```

... the ingressHost should probably be an `am-role-assignment-service` specific one.